internal/cli/ops_manager_diagnose_archive_download.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"io"
+	"os"
+
+	"github.com/mongodb/mongocli/internal/description"
+	"github.com/mongodb/mongocli/internal/flags"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"go.mongodb.org/ops-manager/opsmngr"
+)
+
+type opsManagerDiagnoseArchiveDownloadOpts struct {
+	globalOpts
+	out     string
+	limit   int64
+	minutes int64
+	fs      afero.Fs
+	store   store.ArchivesDownloader
+}
+
+func (opts *opsManagerDiagnoseArchiveDownloadOpts) initStore() error {
+	var err error
+	opts.store, err = store.New()
+	return err
+}
+
+func (opts *opsManagerDiagnoseArchiveDownloadOpts) Run() error {
+	out, err := opts.newWriteCloser()
+	if err != nil {
+		return err
+	}
+
```

Comment:
we are missing something here
```suggestion
	defer out.Close()
```
I saw and I think we are missing this for the OM logs command as well, can you do a small fly by fix there?